### PR TITLE
feat(ui): add wallet connect auth tips

### LINKED ISSUE #5109: Wallet connect sidebar improvements
## Loading 
Is it possible to add a step when you click on polkadot js wallet and when you have to sign the popup from the extension? 
something like waiting for authorization?  - dont know how is it with for example talisman etc. 
image
- Its going to be more friendly for new users - This should prevent thinking that you are connected (and you then close the dialog) once you click on polkadot js and authorization pop ups - you sign it and close the sidebar without choosing account.
image

## Other changes 
- let's make the name of the sidebar weight bold - we are following same visual as on filters - it's clearer what is the content of sidebar and what is the name. 
image

the rest stays the same 
Thank you!

### PR DESCRIPTION
**Pull request by floyd-li**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #5109
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [ ] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=EZiu1PjV2j2JHKxY6mHnFwwCRCoV27uHKQSkKXATSh1srJT&usdamount=0&donation=true)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
image

### CODE REVIEW

File: components/common/ConnectWallet/WalletMenuItem.vue [hunk 1]
```diff
@@ -22,6 +22,33 @@
...
+            stroke="black"
+            stroke-opacity="0.3"
+            stroke-width="2"
+            stroke-linecap="round" />
+          <path
+            d="M10 1.5C14.9706 1.5 19 5.52944 19 10.5"
+            stroke="black"
+            stroke-width="2"
+            stroke-linecap="round" />
+        </svg>
```
Comment (roiLeo) [inline components/common/ConnectWallet/WalletMenuItem.vue:46]: Have you tried one of fontawesome spinner?

File: components/common/ConnectWallet/WalletMenuItem.vue [hunk 2]
```diff
@@ -22,6 +22,33 @@
...
+      </i>
+      <span class="has-text-grey is-size-7 pl-4">
+        Waiting For Authorization
```
Comment (roiLeo) [inline components/common/ConnectWallet/WalletMenuItem.vue:49]: could you add this to translation?

File: components/common/ConnectWallet/WalletMenuItem.vue [hunk 3]
```diff
@@ -105,25 +133,26 @@ const onClickWallet = (wallet: BaseDotsamaWallet): void => {
...
-      walletAccounts.value = data ? data.map(formatAccount) : []
-      localStorage.setItem('wallet', wallet.extensionName)
-    })
-    .catch((e) => {
-      $consola.error('init account error', e)
-    })
+  try {
+    const data = await wallet.getAccounts()
+    walletAccounts.value = data ? data.map(formatAccount) : []
+    localStorage.setItem('wallet', wallet.extensionName)
```
Comment (roiLeo) [inline components/common/ConnectWallet/WalletMenuItem.vue:143]: `useLocalStorage` may help here

File: styles/components/_connect-wallet.scss [hunk 4]
```diff
@@ -94,6 +106,17 @@
           }
         }
       }
+      .auth-tip {
+        padding: 10px 0;
+        margin: 0 2rem;
```
Comment (roiLeo) [inline styles/components/_connect-wallet.scss:111]: why mix `px` & `rem`?

**Review by roiLeo — COMMENTED** [review]
dark mode need some love
Capture d’écran 2023-03-14 à 1 50 11 PM

otherwise wfm

File: styles/components/_connect-wallet.scss (hunk 4 above)
Comment (floyd-li) [reply styles/components/_connect-wallet.scss:111]: i just copied the style of this element `.account-item` in  [_connect-wallet.scss#L97](https://github.com/kodadot/nft-gallery/blob/main/styles/components/_connect-wallet.scss#L97) so the `.auth-tip` i added will keep the same spacing with it.
should i both change these two?

File: components/common/ConnectWallet/WalletMenuItem.vue (hunk 3 above)
Comment (floyd-li) [reply components/common/ConnectWallet/WalletMenuItem.vue:126]: yep, but is it a better idea that this pr do not change the previous usage here?
i can raise another pr to resolve this if it's needed

File: components/common/ConnectWallet/WalletMenuItem.vue (hunk 1 above)
Comment (floyd-li) [reply components/common/ConnectWallet/WalletMenuItem.vue]: sure, i find one :)

File: components/common/ConnectWallet/WalletMenuItem.vue (hunk 3 above)
Comment (roiLeo) [reply components/common/ConnectWallet/WalletMenuItem.vue:126]: Aight

File: components/common/ConnectWallet/WalletMenuItem.vue [hunk 5]
```diff
@@ -22,6 +22,16 @@
         <NeoIcon v-else icon="chevron-right" />
       </div>
     </b-button>
+    <div
+      v-if="isAuth && walletAccounts.length === 0"
+      class="pl-5 pt-2 pb-2 is-flex is-align-items-center auth-tip">
+      <i class="is-flex circle-icon">
+        <i class="fad fa-spinner-third"></i>
+      </i>
```
Comment (roiLeo) [inline components/common/ConnectWallet/WalletMenuItem.vue:30]: Could you please use new component `<NeoIcon />` 🥺

File: styles/components/_connect-wallet.scss (hunk 4 above)
Comment (roiLeo) [reply styles/components/_connect-wallet.scss:111]: you can safely remove `padding: 10px 0;` since we use `pl-5 pt-2 pb-2`

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
pay 40 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $40 @ 36.76 USD/KSM ~ 1.088 $KSM
🧗 [EZiu1PjV2j2JHKxY6mHnFwwCRCoV27uHKQSkKXATSh1srJT](https://kusama.subscan.io/account/EZiu1PjV2j2JHKxY6mHnFwwCRCoV27uHKQSkKXATSh1srJT)
🔗 [0x9c9de3acc7fb1bae5d4c35113fc6e3aae829dde97f7fc6a372f7e3942a0ce81a](https://kusama.subscan.io/extrinsic/0x9c9de3acc7fb1bae5d4c35113fc6e3aae829dde97f7fc6a372f7e3942a0ce81a)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**